public/app/features/integrated-alerting/components/Alerts/Alerts.tsx

```diff
@@ -1,4 +1,124 @@
-import React, { FC } from 'react';
-import { AlertsTable } from './AlertsTable';
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { useStyles, useTheme } from '@grafana/ui';
+import { logger } from '@percona/platform-core';
+import { Column } from 'react-table';
+import { cx } from 'emotion';
+import { Table } from '../Table/Table';
+import { Messages } from '../../IntegratedAlerting.messages';
+import { getStyles } from './Alerts.styles';
+import { Alert, AlertStatus } from './Alerts.types';
+import { formatAlerts, getSeverityColors } from './Alerts.utils';
+import { AlertsService } from './Alerts.service';
+import { AlertRuleSeverity } from '../AlertRules/AlertRules.types';
+import { AlertsActions } from './AlertsActions';
 
-export const Alerts: FC = () => <AlertsTable />;
+const { noData, columns } = Messages.alerts.table;
+const {
+  activeSince: activeSinceColumn,
+  labels: labelsColumn,
+  lastNotified: lastNotifiedColumn,
+  severity: severityColumn,
+  state: stateColumn,
+  summary: summaryColumn,
+  actions: actionsColumn,
+} = columns;
+
+export const Alerts: FC = () => {
+  const style = useStyles(getStyles);
+  const theme = useTheme();
+  const [pendingRequest, setPendingRequest] = useState(false);
+  const [data, setData] = useState<Alert[]>([]);
+  const severityColors = useMemo(() => getSeverityColors(theme), [theme]);
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: summaryColumn,
+        accessor: 'summary',
+        width: '30%',
+      } as Column,
+      {
+        Header: severityColumn,
+        accessor: ({ severity, status }: Alert) => (
+          <span
+            className={cx({
+              [style.getSeverityStyle(severityColors[severity as AlertRuleSeverity])]: status !== AlertStatus.SILENCED,
+            })}
+          >
+            {severity}
+          </span>
+        ),
+        width: '5%',
+      } as Column,
+      {
+        Header: stateColumn,
+        accessor: 'status',
+        width: '5%',
+      } as Column,
+      {
+        Header: labelsColumn,
+        accessor: ({ labels }: Alert) => (
+          <div className={style.labelsWrapper}>
+            {labels.map(label => (
+              <span key={label} className={style.label}>
+                {label}
+              </span>
+            ))}
+          </div>
+        ),
+        width: '40%',
+      } as Column,
+      {
+        Header: activeSinceColumn,
+        accessor: ({ activeSince }: Alert) => <>{activeSince ? activeSince : null}</>,
```

Comment:
Is the fragment tag (`<>`) needed here?

Reply:
Hm, right, will just call the accessor straight away, it can be empty like the other tables